pegasus/sites.v3/code.org/views/educate_box.haml

```diff
@@ -1,4 +1,5 @@
 -color ||= "#59b9dc"
+-item[:button_s] ||= 'Learn more'
```

Comment:
Since they were all the same I thought it'd be unnecessarily repetitive to put 'Learn more' explicitly for each box. If we were to be purist about it I would have put all the table cleanup/refactoring in a previous PR... I felt like this was related enough to be together.